workshop/exercise-5/index.html

```diff
@@ -1,16 +1,27 @@
 <!DOCTYPE html>
 <html>
-
-<body>
-    Things I like
-    <p></p>
+  <body>
+    <h1>Things I like</h1>
+    <p>
+      Lorem ipsum dolor sit amet consectetur, adipisicing elit. Obcaecati
+      corrupti quibusdam laudantium repellendus fuga incidunt eligendi cumque
+      nam ea, minus dolor mollitia voluptas id non iste! Obcaecati tenetur
+      sapiente nemo.
+    </p>
+    <h3>Additionally</h3>
     <ul>
-        <li>Javascript</li>
+      <li>Javascript</li>
+      <li>Brownies</li>
+      <li>Dad jokes</li>
     </ul>
     <h3>Fun facts</h3>
-    <p>languages are spoken today</p>
-    <img />
-    <p></p>
-</body>
-
-</html>
\ No newline at end of file
+    <p>7,111 languages are spoken today</p>
+    <img src="/workshop/exercise-5/imgs/languages.png" alt="languages" />
```

Comment:
currently it will look for a workshop folder in your present directory, which does not exist so we give it the path from current folder i.e: src="img/languages.png" should solve it